mpas_analysis/analysis_task_template.py

```diff
@@ -0,0 +1,217 @@
+'''
+This is an example analysis task to be used as a template for new tasks.
+It should be copied into one of the component folders (`ocean`, `sea_ice`,
+`land_ice`, etc.) and modified as needed.
+
+Specific parts or variable names that need to be modified by the developer are
+explicitly indicated. All the others are to be left unchanged.  Anything in
+comments or docstrings in triangle brackets <> indicates that it needs to be
+replaced with the appropriate variable name, text, etc. for the task.
+
+Instructions for creating a new analysis task:
+1. create a new task by copying this file to the appropriate folder (ocean,
+   sea_ice, etc.) and modifying it as described below.  Take a look at
+   mpas_analysis/shared/analysis_task.py for additional guidance.
+2. modify config.default (and possibly any machine-specific config files in
+   configs/<machine>)
+3. import new analysis task in mpas_analysis/<component>/__init__.py
+4. note, no changes need to be made to mpas_analysis/shared/analysis_task.py
+5. if necessary, update variable maps in
+   mpas_analysis/shared/variable_namelist_stream_maps/<component>_maps.py
+6. add new analysis task to run_analysis.py under build_analysis_list:
+      analyses.append(<component>.MyTask(config, myArg='argValue'))
+   will add a new object of the MyTask class to a list of analysis tasks
+   created in build_analysis_list.  Later on in run_analysis, it will first
+   go through the list to make sure each task needs to be generated
+   (by calling check_generate, which is defined in AnalysisTask), then, will
+   call setup_and_check on each task (to make sure the appropriate AM is on
+   and files are present), and will finally call run on each task that is
+   to be generated and is set up properly.
+
+Don't forget to remove this docstring. (It's not needed.)
+
+Authors
+-------
+Xylar Asay-Davis
+'''
+
+# import python modules here
+
+# import mpas_analysis module here (those with relative paths starting with
+# dots)
+from ..shared.analysis_task import AnalysisTask
+
+
+# Everywhere in this template, change MyTask to the name of your task, starting
+# with a capital letter and using mixed case but no underscores.  Leave
+# "AnalysisTask" as it is.  This indicates that MyTask has AnalysisTask as its
+# parent, "inhereting" several variables and methods (functions) from it that
+# can be helpful in initializing, checking whether to perform analysis and
+# performing the analysis.  See AnalysisTask in shared/analysis_task.py
+class MyTask(AnalysisTask):  # {{{
+    '''
+    <Briefly describe the analysis task here.  Just a 1-2 sentence description
+    of what the task does should do.>
```

Comment:
I don't see a typo.  Perhaps "should do" is confusing so I will change it to "should be sufficient".